### DISCUSSION TITLE
About Multilingual Pages

### DISCUSSION THREAD

**Discussion by cortix** [mmistakes/minimal-mistakes#4036] [Q&A]
I'm currently trying to understand how google detects multilingual pages. My current blog has both Turkish and English language options. But on my English page, the translations of all the content on my Turkish page are not available. Google provides an explanation for this situation.

> **Here are the most common mistakes with hreflang usage:**
> 
> Missing return links: If page X links to page Y, page Y must link back to page X. If this is not the case for all pages that use hreflang annotations, those annotations may be ignored or not interpreted correctly. For example, considering this link on https://de.example.com/index.html:
> 
> `<link rel="alternate" hreflang="en-gb" href="https://en-gb.example.com/index.html" />`
>
> You must also have a hreflang link on https://en-gb.example.com/index.html that points back to the de version of the content:
> 
> `<link rel="alternate" hreflang="de" href="https://de.example.com/index.html" />`

At first, I had added the `hreflang` tag only in `_includes/head/custom.html`. 

```
<link rel="alternate" hreflang="en" href="https://en.hasancelik.org"/>
<link rel="alternate" hreflang="tr" href="https://hasancelik.org"/>
```

But as I understand, this means that all my posts have an English equivalent. But not all my posts have English translation. My question is; Is it possible to add `hreflang` only to posts with translation instead of `_includes/head/custom.html`?

Reference: [Google about localized versions of your page](https://developers.google.com/search/docs/specialty/international/localized-versions#common-mistakes)

**Comment by nenge123** [comment]
page a | a_ae_ar
`<link rel="alternate" href="index.html" hreflang="en-US" />`
`<link rel="alternate" href="{{- %link lang/index_ae_AR.md -%}}" hreflang="ar-AE" />`


page b
`<link rel="alternate" href="index.html" hreflang="en-US" />`
`<link rel="alternate" href="{{- %link lang/index_zh_CN.md -%}}" hreflang="zh-CN" />`


```md
---
lang: ['zh-CN','ar-AE']
---
```

```
{{%  for  lang in page.lang %}}
<link rel="alternate" href="{{- %link lang/index_zh_CN.md -%}}" hreflang="{{%-  lang -%}}" />`
{% endfor %}
```
